### DISCUSSION TITLE
how to set main country page the same as in browser

### DISCUSSION THREAD

**Discussion by omega3** [FreeTubeApp/FreeTube#5067] [Q&A] [answered]
When I open youtube in web browser it shows the same as when clicked on Home icon and movies mostly from local / proper to location channels. In freetube I see only Tranding (I set my location as region) and Most popular. But movies are not the same as in browser's youtube Main page. In Most popular in freetube I have only US movies. There is no indivious instance in my country. I set Local API.

**Comment by absidue** [comment, marked as answer]
FreeTube doesn't get any information from YouTube's home page.

Trending is the same as the trending page on the website: https://www.youtube.com/feed/trending

Most Popular are the most popular videos on the Invidious instance that you have set FreeTube to use (if you haven't manually set one, FreeTube automatically picks a random Invidious instance from Invidious' list of instances).
